Formula/cockroach.rb

```diff
@@ -19,6 +19,8 @@ class Cockroach < Formula
   depends_on "make" => :build
   depends_on "xz" => :build
 
+  patch :DATA
```

Comment:
The PR patch can't be used directly, since the paths don't quite line up. I've moved the patch to a gist, however, and added a few sentences describing the situation.